### DISCUSSION TITLE
Using other credentials to db causes error

### DISCUSSION THREAD

**Discussion by aleksejssaburovs92** [directus/directus#5153] [Archive]
change in env file variables
DB_USER
DB_PASSWORD
(User has access to DB)

start project with docker-compose up 
Actual result:

```
error: create table "directus_collections" ("collection" varchar(64), "icon" varchar(30), "note" text, "display_template" varchar(255), "hidden" boolean not null default '0', "singleton" boolean not null default '0', "translations" json, "archive_field" varchar(64), "archive_app_filter" boolean not null default '1', "archive_value" varchar(255), "unarchive_value" varchar(255), "sort_field" varchar(64)) - relation "directus_collections" already exists
```

<img width="1417" alt="Screenshot 2021-04-19 at 19 13 19" src="https://user-images.githubusercontent.com/12848012/115268798-51ab6580-a143-11eb-8e5c-1490ff8b2bee.png">

Expected result:
directus    | 16:12:55 ✨ Initializing bootstrap...
directus    | 16:12:56 ✨ Database already initialized, skipping install

<img width="617" alt="Screenshot 2021-04-19 at 19 13 59" src="https://user-images.githubusercontent.com/12848012/115268882-6a1b8000-a143-11eb-925f-d2420f0d3d9b.png">

Version: v9.0.0-rc.59
Installation and running: Docker (docker-compose)
DB: pg

**Comment by rijkvanzanten** [comment]
What does your docker-compose file look like? What are the permissions for your database user? Could it be that this user doesn't have read access to the schema information, making it impossible for directus to "see" the existing tables?
